mobile/src/board/entity.tsx

```diff
@@ -0,0 +1,60 @@
+export enum GameState {
```

Comment:
This file only contains types. The convention is to call it `types.ts`(no need for `.tsx` as there is no JSX)